mirrord/config/src/feature/copy_target.rs

```diff
@@ -0,0 +1,33 @@
+use mirrord_config_derive::MirrordConfig;
+use schemars::JsonSchema;
+
+use crate::{
+    config::{from_env::FromEnv, source::MirrordConfigSource, ConfigContext, ConfigError},
+    util::MirrordToggleableConfig,
+};
+
+#[derive(MirrordConfig, Default, PartialEq, Eq, Clone, Debug)]
+#[config(map_to = "CopyTargetFileConfig", derive = "JsonSchema")]
+#[cfg_attr(test, config(derive = "PartialEq, Eq"))]
+pub struct CopyTargetConfig {
+    #[config(env = "MIRRORD_COPY_TARGET_ENABLED")]
+    pub enabled: bool,
+
+    #[config(env = "MIRRORD_COPY_TARGET_NAME")]
+    pub name: Option<String>,
+}
+
+impl MirrordToggleableConfig for CopyTargetFileConfig {
+    fn disabled_config(context: &mut ConfigContext) -> Result<Self::Generated, ConfigError> {
+        let enabled = FromEnv::new("MIRRORD_COPY_TARGET_ENABLED")
```

Comment:
that's a bit weird, why not to rely on the copy target existing?

Reply:
I intend for the use of target and the `name` is for more complex if you want to connect to an already running copy

Reply:
It's not a requirement atm, maybe drop it to make it simpler?

Reply:
?